pkg/io/video/scale_test.go

```diff
@@ -247,6 +525,103 @@ func TestScale(t *testing.T) {
 	}
 }
 
+func TestScaleFastBoxSampling(t *testing.T) {
+
```

Comment:
```suggestion
```

Reply:
Removed in 466d14b1737a701bc31500700f8ef6f51147d76b.